src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -675,14 +675,10 @@ private static synchronized TimeZone setDefaultZone() {
         // platform to Java time zone ID mapping.
         if (zoneID == null || zoneID.isEmpty()) {
             String javaHome = StaticProperty.javaHome();
-            try {
                 zoneID = getSystemTimeZoneID(javaHome);
```

Comment:
Nit: align the indentation. Also, `javaHome` variable can be eliminated.

Reply:
Thanks for catching that, fixed